Gini-iOS-SDK/GINIURLSessionDelegate.m

```diff
@@ -7,67 +7,26 @@
 //
 
 #import "GINIURLSessionDelegate.h"
+#import <TrustKit/TrustKit.h>
 
 @implementation GINIURLSessionDelegate {
-    NSArray<NSString *> *_nsCertificatePaths;
 }
 
-+ (instancetype)urlSessionDelegateWithCertificatePaths:(NSArray<NSString *> *)certificatePaths {
-    return [[self alloc] initWithNSURLSessionDelegate:certificatePaths];
-}
-
-- (instancetype)initWithNSURLSessionDelegate:(NSArray<NSString *> *)certificatePaths {
-    self = [super init];
-    if (self) {
-        _nsCertificatePaths = certificatePaths;
-    }
-    return self;
++ (instancetype)urlSessionDelegate {
+    return [self alloc];
```

Comment:
Why not `init`?

Reply:
Because in this case is the same, since it is the default initializer. For me there isn't a big difference.